config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
-  resources :books, only: [:show]
+  resources :books, only: [:show] do 
+    resources :reviews
+  end
+  resources :users
```

Comment:
Careful to pay close attention to your user stories. Each story asks for something on the book show, that should be the only route you need.